packages/camera/tizen/src/camera_device.cc

```diff
@@ -345,20 +346,19 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
               current_packet_ = nullptr;
               return nullptr;
             }
-            flutter_desktop_gpu_buffer_->buffer = surface;
-            flutter_desktop_gpu_buffer_->width = width;
-            flutter_desktop_gpu_buffer_->height = height;
-            flutter_desktop_gpu_buffer_->release_callback =
-                [](void *release_context) {
-                  CameraDevice *cd = (CameraDevice *)release_context;
-                  cd->ReleaseMediaPacket();
-                };
-            flutter_desktop_gpu_buffer_->release_context = this;
-            return flutter_desktop_gpu_buffer_.get();
+            gpu_surface_->handle = surface;
+            gpu_surface_->width = width;
+            gpu_surface_->height = height;
+            gpu_surface_->release_callback = [](void *release_context) {
+              CameraDevice *cd = (CameraDevice *)release_context;
+              cd->ReleaseMediaPacket();
+            };
+            gpu_surface_->release_context = this;
+            return gpu_surface_.get();
           }));
   texture_id_ =
       registrar_->texture_registrar()->RegisterTexture(texture_variant_.get());
-  flutter_desktop_gpu_buffer_ = std::make_unique<FlutterDesktopGpuBuffer>();
+  gpu_surface_ = std::make_unique<FlutterDesktopGpuSurfaceDescriptor>();
```

Comment:
`FlutterDesktopGpuSurfaceDescriptor` is a simple struct type. Should this value managed as a unique pointer?

Reply:
> I mean, the value doesn't need to be a member of the class.

Did you mean that we don't need to have a pointer of `FlutterDesktopGpuSurfaceDescriptor` as member?
I'd appreciate it if you could give me a little more detail.

Reply:
Ah, never mind. I thought `GpuSurfaceTexture`'s `ObtainDescriptorCallback` returns `FlutterDesktopGpuSurfaceDescriptor` by value so we can initialize an instance of `FlutterDesktopGpuSurfaceDescriptor` inside the callback.

Still we can convert `gpu_surface_` into a plain struct (not unique pointer) but it shouldn't matter much.

Reply:
thanks! :)